tests/test_core.py

```diff
@@ -6280,11 +6280,10 @@ def check(out, err):
 
   ### Integration tests
 
-  @sync
   def test_ccall(self):
     self.emcc_args.append('-Wno-return-stack-address')
     self.set_setting('EXTRA_EXPORTED_RUNTIME_METHODS', ['ccall', 'cwrap'])
-    create_test_file('post.js', '''
+    self.add_pre_run('''
```

Comment:
i don't think this will work, `preRun` is for stuff like filesystem preloading that happens before the runtime is ready yet.

Reply:
At yes, we should add addPostRun I guess?

Reply:
Yeah, makes sense to add a postRun option for this I think.